quokka/core/content/models.py

```diff
@@ -230,13 +230,13 @@ def metadata(self):
         # TODO: get metadata from database
         # TODO: implement libratar/gravatar
         # return {
-            # 'cover': 'foo',
-            # 'author_gravatar': 'http://i.pravatar.cc/300',
-            # 'about_author': 'About Author',
-            # 'translations': ['en'],
-            # 'og_image': 'foo',
-            # 'series': 'aa',
-            # 'asides': 'aaa'
+        # 'cover': 'foo',
```

Comment:
Consider remover these comments

Reply:
already fixed in another pr => https://github.com/rochacbruno/quokka/pull/680